Sources/SwiftToolchain/ToolchainManagement.swift

```diff
@@ -291,4 +292,19 @@ extension FileSystem {
 
     return result.sorted()
   }
+  
+  public func fetchLocalSwiftVersion() throws -> String? {
+    guard isFile(swiftVersionPath), let version = try readFileContents(swiftVersionPath)
+      .validDescription?
+      // get the first line of the file
+      .components(separatedBy: CharacterSet.newlines).first
+    
```

Comment:
**nitpick:** no need for the empty line here, which would be consistent with the rest of `guard` statements in the codebase
```suggestion
```